sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/rules/QueryExecutionMetering.scala

```diff
@@ -85,10 +85,11 @@ case class QueryExecutionMetering() {
       s"$ruleName $runtimeValue $numRunValue"
     }.mkString("\n", "\n", "")
 
+    val format = new java.text.DecimalFormat("#,##0.#########")
     s"""
        |=== Metrics of Analyzer/Optimizer Rules ===
        |Total number of runs: $totalNumRuns
-       |Total time: ${totalTime / NANOS_PER_SECOND.toDouble} seconds
+       |Total time: ${format.format(totalTime / NANOS_PER_SECOND.toDouble)} seconds
```

Comment:
Just use `String.format`?

Reply:
> Just use `String.format`?

`String.format("%.9f, args)` can't handle  extra 0 in the end. 
Such as `String.format("%.9f, (100/ 1000000000D))` = `0.0000000100`

Reply:
How about this? https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/ui/UIUtils.scala#L50

Reply:
> How about this? https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/ui/UIUtils.scala#L50

`UIUtils.formatDuration(totalTime / NANOS_PER_MICROS)`
Use this method will lose  precision， if you don's mind, it's ok.

Reply:
@maropu  Change to `String.format`